plugins.d/Lets_Encrypt/dns_01.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python3
+import sys
+import subprocess
+from subprocess import PIPE
+from os import path
+from shutil import which
+import re
```

Comment:
How should I go about creating a changelog given this is not an appliance, but a TKL component?

Reply:
Good point. The way that it's meant to work is create a new file in `docs/` called `RelNotes-x.y.x.txt` (where `x.y.z` is the version number). But I've been slack and haven't done that since v1.0.0 release a few years ago...

Please do do that though so there's record of all this work you've done. Also, let's make the release for your updates `v2.1.0` (the current version is `2.0.5` and IMO these changes add significant functionality, enough to warrant a minor version bump IMO.

Noe, I'll need to take care of that when I build (I'll tag the repo once it's all merged in and then the version number of the package is autogenerated from the tag).